index.bs

```diff
@@ -10600,129 +10606,162 @@ can restart executing this algoritm if needed.
 	until {{OfflineAudioContext/length}} frames have been rendered.
 </div>
 
+The system level audio callback is also queued as a task in the
+<a href="#control-message-queue">control message queue</a>. The UA MUST perform
+the following sequence to process render quanta to fulfill such task by filling
+up the requested buffer size. Along with the primary message queue, the
+<a>rendering thread</a> has another task queue for microtasks for any microtask
+operation such as resolution of {{Promise}}s in the {{AudioWorkletGlobalScope}}.
+
 <div algorithm="rendering a graph">
-	1. Set the internal slot <dfn attribute
-		for="BaseAudioContext">[[current frame]]</dfn> of the
-		{{BaseAudioContext}} to 0.
-	1. Let <em>Q<sub>rendering</sub></em> be an empty <a>control message
-		queue</a>. <a>Atomically</a> <a>swap</a>
-		<em>Q<sub>rendering</sub></em> and <em>Q</em>
+  1. Prepare the rendering.
+
+		1. Let <em>Q<sub>rendering</sub></em> be an empty <a>control message
+			queue</a>. <a>Atomically</a> <a>swap</a>
+			<em>Q<sub>rendering</sub></em> and <em>Q</em>.
```

Comment:
Not your fault, but Q seems undefined here.

Reply:
I didn't touch this and want to fix this in a follow up PR if needed.